src/devtools/devtools.tsx

```diff
@@ -245,14 +265,14 @@ export function ReactQueryDevtools({
           <Button
             type="button"
             aria-label="Close React Query Devtools"
-            {...otherCloseButtonProps}
-            onClick={() => {
+            {...(otherCloseButtonProps as any)}
```

Comment:
Same reason as above here as well, it was giving the same error as above for the `ref` attribute

Reply:
weird - I couldn't find out why that is. What you can do however is:

```
{...(otherCloseButtonProps as unknown)}
```

spreading `any` onto the Button will make all the other props _not_ typecheck anymore, so it's kind of a "leaking" any. with unknown, all other props will still type check, so it's at least non-invasive.

Reply:
✅